### ISSUE TITLE
win64 Pre-built binary only works for Electron v1.4.*

### ISSUE THREAD

**Issue by Cleod9** [greenheartgames/greenworks#129]
It seems the pre-built Electron binaries (At least for Win64) were not built with the same node_version as Electron 1.6.* (which internally runs under node 7.*)
See: https://github.com/electron/electron/blob/v1.6.1/.node-version

Example error with Electron v1.5.*:
```
Uncaught Error: The module '\\?\C:\NintendoWebFramework\Projects\YeahJamFuryU\app\lib\greenworks-win64.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 50. This version of Node.js requires
NODE_MODULE_VERSION 51. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or`npm install`).
```
(And in the case of Electron v1.6.1, it complains about requiring NODE_MODULE_VERSION 53 instead of 51)

Works fine if i install electron 1.4.*.

It's possible that there are issues with the other pre-built binaries as well, but I don't have the devices/resources to be able to confirm this. Definitely something to look into.

Also personally I'd suggest that the prebuilt binaries target Electron 1.5.* instead since its underlying node version has LTS, but at the very least it should support anything beyond Electron 1.4

**Comment by hokein** [comment]
From your message, looks like you are using the prebuilt greenwork binary for electron v1.4.X. Can you double-check whether you download the correct prebuilt binary?

I can't reproduce your issues with the prebuilt `greenworks-v.0.9.0-electron-v1.6.1-win-64` binary on electron v1.6.1.  It works properly on my machine.

> Also personally I'd suggest that the prebuilt binaries target Electron 1.5.* instead since its underlying node version has LTS, but at the very least it should support anything beyond Electron 1.4

For electron, it is not always correct. There is no guarantee that the node addon built for `1.x.*` is compatible for `1.x+1.*`, which depends on whether the next version upgrades the node etc. But the prebuilt addon should be compatible for all 1.x.* versions.

BTW, we don't provide prebuilt greenworks binaries as Electron 1.5.* doesn't have any stale releases.

**Comment by Cleod9** [comment]
@hokein Thanks for the response, it turns out that I actually ran into a problem that I edited just now in another issue (No. 5):

https://github.com/greenheartgames/greenworks/issues/128

I thought I had updated my libs folder with the proper prebuilt libs, but it turns out the win64 version was the one from the `greenworks` package on NPM (which is not managed by you guys). I've removed all traces from that cursed fork on NPM and everything is now up to date. 

Anyway, I'm good with this so I'll close it
